src/apiCalls.js

```diff
@@ -15,80 +17,114 @@ export const fetchData = () => {
     });
 };
 
-
-export const postProject = (userId, title, description, color, deadline) => {
-  return fetch(
+export const postProject = (
+  userId,
+  title,
+  description,
+  color,
+  deadline,
+  dispatch
+) => {
+  fetch(
     `https://analog-be-18680af1ea7c.herokuapp.com/api/v1/users/${userId}/projects/`,
-    
-  {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(
-      {
+
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
         title: title,
         description: description,
         color: color,
         deadline: deadline,
-    })
-  })
-  .then((response) => {
-    if(!response.ok) {
-      throw new Error(`Failed to POST`)
+      }),
     }
-  })
-  .catch((error) => {
-    console.error(`Error in Network Request`, error);
-  });
+  )
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(`Failed to POST`);
+      }
+      fetchData().then((data) => {
+        dispatch(setUserData(data.data));
+      });
+    })
```

Comment:
It looks like most of this PR is some stylistic cleanup, but I noticed these lines added to a few places. Can you share what this is doing and why you added it?

Reply:
This line is making the GET request following a POST or PATCH request in order to GET the updated user data and set the global state (redux) to the appropriate updated state via dispatching the reducer called setUserData.

Reply:
Neat! Thanks for the info.